### DISCUSSION TITLE
Java API?

### DISCUSSION THREAD

**Discussion by siddhsql** [speedb-io/speedb#673] [General]
Does speedb have a Java API? where can I get the details?

**Comment by Guyme** [comment]
Yes, Speedb has Java API - same as Rocksdb, you can find details here 
https://javadoc.io/doc/org.rocksdb/rocksdbjni/latest/index.html
We also have a Speedb maven repo here 
https://mavenlibs.com/maven/dependency/io.github.speedb-io/speedbjni

**Reply by siddhsql** [reply]
Thanks Guyme. Got it!
